### PR TITLE
Remove attempt to use wizard on manage ACLs

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
As reported by @christianwach we are currently seeing smarty failures with php3 on the Manage ACLs screen due to it trying to add wizard navigation at the tpl layer - this just removes that on the basis it adds marginal value to a screen that is already up for replacement - also it didn't really seem to be doing what it said on the box - ie the UI seemed the same after I ripped out the code as before....

Before
----------------------------------------
image

After
----------------------------------------
Loads, no wizard header

Technical Details
----------------------------------------
We should back port to 5.69 as that was where it was reported

Comments
----------------------------------------

### CODE REVIEW

File: templates/CRM/ACL/Header.tpl [hunk 1]
```diff
@@ -13,40 +13,3 @@
 <div class="help">
   <p>{ts 1=$docLink}ACLs allow you to control access to CiviCRM data. An ACL consists of an <strong>Operation</strong> (e.g. 'View' or 'Edit'), a <strong>set of data</strong> that the operation can be performed on (e.g. a group of contacts), and a <strong>Role</strong> that has permission to do this operation. Refer to the %1 for more info.{/ts}</p>
 </div>
-
-{php}
-  $currentStep = $this->getTemplateVars('step');
-  $wizard = [
-    'style' => ['barClass' => '', 'showTitle' => FALSE],
```
Comment (totten) [inline templates/CRM/ACL/Header.tpl:20]: I signed up for a bar class, but my style still came out empty. I probably studied under the wrong teacher.

Comment (eileenmcnaughton) [reply templates/CRM/ACL/Header.tpl:20]: @totten you got shorter ^^

Comment (totten) [reply templates/CRM/ACL/Header.tpl:20]: @eileenmcnaughton For that hair, it's an even trade.

Comment (eileenmcnaughton) [reply templates/CRM/ACL/Header.tpl:20]: lol

**Comment by totten** [comment]
> ...this just removes that on the basis it adds marginal value to a screen that is already up for replacement - also it didn't really seem to be doing what it said on the box - ie the UI seemed the same after I ripped out the code as before...

I agree. There is a very subtle difference. (The visual effect of the empty/non-functioning wizard was akin to an extraneous `<hr/>` -- it just appears as a pointless line.) So it's not adding anything. And most pages which hit this file already have replacements in `civicrm_admin_ui`.

**Pull request merged**